src/test/data/JsonSerializableMenuTest/typicalFoodMenu.json

```diff
@@ -64,12 +64,28 @@
     "restrictions": "Halal"
   } ],
   "purchases" : [ {
-    "purchasedFoodName" : "Ji Fan",
-    "purchasedFoodPrice" : "3.99",
+    "purchasedFood" : {
+      "name" : "Carbonara",
+      "price" : "5.00",
+      "description" : "Italian noodle with cheese and meat",
+      "category": "Italian",
+      "tagged" : [ "Noodle" ],
+      "location" : "The Deck",
+      "openingHours" : "0800 1700",
+      "restrictions" : "Contains dairy and pork"
+    },
     "timeOfPurchaseInMillisSinceEpochString" : "1570976664361"
```

Comment:
Might be good to save it in a more readable format. But this is a small issue since no one reads this file except us.